packages/core/src/types/module-builder.ts

```diff
@@ -193,6 +323,32 @@ export interface IgnitionModuleBuilder {
     options?: ContractAtOptions
   ): ContractAtFuture<AbiT>;
 
+  /**
+   * Reads an event argument from a contract.
```

Comment:
Maybe a follow up line, on why you want to do that, i.e. so you can use the value in subsequent futures as a parameter.